### PR TITLE
Add Generic Approval Voting Templates

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
Adds templates for approval voting with and without majority requirement.

Based on example polls approved by DUX here: https://github.com/makerdao-dux/community/tree/dux/goerli-polls/governance/polls/poll-parameters

### CODE REVIEW

File: governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    options: [1,2,3]
+    abstain: [0]
+  victory_conditions:
+    - { 
+        type: 'and', 
+        conditions: [
+          { type : approval },
+          { type: majority, percent: 50 },
+        ]
+      }
```
Comment (LongForWisdom) [inline governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:17]: Can rewrite as if you want:
  ```
victory_conditions:
    - type: and
      conditions:
          - { type : approval }
          - { type: majority, percent: 50 }
    - { type : default, value : $StatusQuoOrRejectOptionNumber }
```

Comment (LongForWisdom) [reply governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:17]: Slightly cleaner.

File: governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Poll:$GenericChange - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the $poll_source. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is an approval vote.**
+- **You may vote for multiple options.**
+- **You should select all the options that you would support in an executive vote.**
```
Comment (LongForWisdom) [inline governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:34]: 'No Change' needs to always be an option for these. Makes this a little misleading. We might have the same problem elsewhere though.

Comment (LongForWisdom) [reply governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:34]: (Unless we're disregarding the majority condition)

File: governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+The community can vote in this poll to express support or opposition to the following changes:
+* $GenericChangeA
+* $GenericChangeB
+* $GenericChangeC
+
+Please review the discussion [thread]($discussion_link) to help inform your position before voting.
+
+## Outcomes
+
+**The winner of this vote requires an overall majority of votes to support it. The non-`Abstain` option with the most votes AND the support of a majority of votes is deemed the winner and the following actions will be taken:**
```
Comment (LongForWisdom) [inline governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:49]: It's possible for approval votes to have multiple winning options, and for it to be valid to implement more than one. Something like:
"How should we fix the peg?"
A: raise rates
B: print unbacked DAI
C: have a party
Those aren't mutually exclusive outcomes, and all >50% should be implemented.

Comment (LongForWisdom) [reply governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:49]: Probably those sorts of votes should have a separate template. approval-mutex vs approval-nonex

File: governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows.
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
```
Comment (LongForWisdom) [inline governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:54]: Not accurate? "If the votes for the 'No' option equal or exceed the votes for all other options, or if no other option reaches a majority then no further action will be taken at this time.**"

Alternatively:

**If the condition above is not met, then no further action will be taken at this time.**

File: governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+Please review the discussion [thread]($discussion_link) to help inform your position before voting.
+
+## Outcomes
+
+**The non-'Abstain' option with the most votes is deemed the winner and the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows.
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
```
Comment (LongForWisdom) [inline governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md:45]: The assumption with this poll type is that there isn't a 'no' and a 'yes' option and that it's more of a choice between two options and that status-quo isn't available. So, given that, this doesn't make the most sense.

File: governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
+---
+title: $GenericChange - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition to $GenericChange.
+discussion_link: $discussion_link
+parameters:
+  input_format:
+    type: choose-free
+    options: [1,2]
```
Comment (LongForWisdom) [inline governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md:8]: Approval-without-majority with two options is just a plurality poll. Voting for both or neither is a non-vote, voting for one and not the other = plurality.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments

File: governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md (hunk 5 above)
Comment (patrick-j-govalpha) [reply governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md:45]: fixed

File: governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md (hunk 6 above)
Comment (patrick-j-govalpha) [reply governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md:8]: This was an oversight. I have added a third option to the template.

File: governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md (hunk 1 above)
Comment (patrick-j-govalpha) [reply governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md:17]: Will this definitely work? You have a greater grasp of yaml than I do. I know that the way I've written it worked in the Goerli polls we did

**Pull request merged**